src/reputation/migrations/0062_auto_20220926_1846.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2 on 2022-09-26 18:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Is this migration also related to your branch? Seems like it may not

Reply:
You're right it's not, it came from https://github.com/ResearchHub/researchhub-backend/pull/1151 and I think it's appearing int his PR because I merged in master after pushing this PR initially and pushed again.

This is what I had in mind when I said "It looks a bit messy to view but I found that you can pick my commits in the Files changed tab so it doesn’t show diffs for the stuff merged in from master." Alternatively, as the owner of this PR, I can save you that trouble if I rebase my commits on top of master and force push to this PR.

Reply:
I think rebasing on master and force pushing is good.

Reply:
agree with rebasing on top of master and force pushing, this way we know exactly what's inside of the PR

Reply:
ok, i'll do that henceforth. thanks!